test/Microsoft.TestPlatform.AcceptanceTests/ExecutionTests.cs

```diff
@@ -94,16 +96,58 @@ public void WorkingDirectoryIsSourceDirectory(string runnerFramework, string tar
         [CustomDataTestMethod]
         [NET46TargetFramework]
         [NETCORETargetFramework]
-        public void ShouldGetErrorMessageOnTesthostExitWithError(string runnerFramework, string targetFramework, string targetRuntime)
+        public void StackOverflowExceptionShouldBeLoggedToConsoleAndDiagLogFile(string runnerFramework, string targetFramework, string targetRuntime)
         {
             AcceptanceTestBase.SetTestEnvironment(this.testEnvironment, runnerFramework, targetFramework, targetRuntime);
 
+            var diagLogFilePath = Path.Combine(Path.GetTempPath(), $"std_error_log_{Guid.NewGuid()}.txt");
+            File.Delete(diagLogFilePath);
+
             var assemblyPaths =
                 this.BuildMultipleAssemblyPath("SimpleTestProject3.dll").Trim('\"');
             var arguments = PrepareArguments(assemblyPaths, this.GetTestAdapterPath(), string.Empty, this.FrameworkArgValue);
-            arguments = string.Concat(arguments, " /tests:ExitWithStdErrorMessageTest");
+            arguments = string.Concat(arguments, " /tests:ExitwithUnhandleException");
+            arguments = string.Concat(arguments, $" /diag:{diagLogFilePath}");
```

Comment:
We shouldn't enable `diag` for this scenario. This should ensure everything shows up in console and VS without any special setting.